index.html

```diff
@@ -0,0 +1,228 @@
+            <button class="burger"><span></span></button>
+          </div>
+        </div>
+        <nav class="mobile-nav">
+          <ul class="mobile-nav__list">
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Who We Are</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">What We Do</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Travel With Us</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Contact Us</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+    <div class="travel__why">
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
+            <div class="container__why">
+              <h2 class="travel-text__bigger">Why Choose Us</h2>
+              <h4 class="travel-text__small">
+                Tion and dislike men who are so beguiled and demoralized by
+                desire by the charms of pleasure of the moment, so blinded by
+                desire, that they cannot foresee Ddislike men who are so
+                beguiled and demoralized by the charms of pleasure of the
+                moment, so blinded by desire, that.
+              </h4>
+              <button class="travel-button__explores">Eplore Places</button>
+            </div>
+          </div>
+          <div class="col-5">
+            <img src="img/Image1.png" alt="" />
+            <img class="margeningtotheright" src="img/Image2.png" alt="" />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__steps">
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <img src="img/search.png" alt="" />
+            <h2 class="travel-bigger">Search Places</h2>
+          </div>
+          <div class="col-4">
+            <img src="img/check-circle.png" alt="" />
+            <h2 class="travel-bigger">Select Places</h2>
+          </div>
+          <div class="col-4">
+            <img src="img/thumbs-up.png" alt="" />
+            <h2 class="travel-bigger">Book & Enjoy</h2>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__book1">
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
+            <div class="travel-spain__book1">
+              <h5 class="travel-text__verysmall">Spain</h5>
+              <h3 class="travel-text__middle">Grand Spain Madrid</h3>
+              <h4 class="travel-text__small">
+                But I must explain to you how all this mistaken idea of <br />
+                account to denouncing pleasure and praising pain was <br />
+                account born and I account will give you a complete <br />
+                account of the system
+              </h4>
```

Comment:
это параграф поэтому `p`
```suggestion
              <p class="travel-text__small">
                But I must explain to you how all this mistaken idea of <br />
                account to denouncing pleasure and praising pain was <br />
                account born and I account will give you a complete <br />
                account of the system
              </p>
```